### PR TITLE
Update Wasm page for current status after finalized WasmGC encodings

### PR DESCRIPTION
**Pull request by parlough**
Now that Chrome 119 has been released to stable with the finalized encodings and the Flutter `main` channel has been updated to use them, this doc needs an update. 

I've updated the doc to reflect Chrome 119 and Firefox 120 as the new minimums.

I've also completed some additional cleanup and clarified the latest version of the `main`/`master` channel is needed.

Staged: https://flutter-docs-prod--pr9711-fix-oct-26-wasm-upda-ipg2m54p.web.app/platform-integration/web/wasm

### CODE REVIEW

File: src/platform-integration/web/wasm.md
Comment (kevmoo) [inline src/platform-integration/web/wasm.md:3]: Nice!

File: src/platform-integration/web/wasm.md
Comment (kevmoo) [inline src/platform-integration/web/wasm.md:29]: whitespace churn makes it hard to review! 😄

Comment (parlough) [reply src/platform-integration/web/wasm.md:29]: Ah sorry, I updated to closer match our standards. I'll split it into different commits next time!

The main change in the PR is mentioning Chrome 119 and Firefox 120 without flags, and being more explicit about updating to the latest `main`.

File: src/platform-integration/web/wasm.md
Comment (atsansone) [inline src/platform-integration/web/wasm.md:56]: This could be written in a more concise way.


```suggestion
To verify if your Flutter install uses the `master` channel,
```

Verify and correctly are redundant.

File: src/platform-integration/web/wasm.md
Comment (atsansone) [inline src/platform-integration/web/wasm.md:59]: **issue:** Wasm is always initial caps.

File: src/platform-integration/web/wasm.md
Comment (atsansone) [inline src/platform-integration/web/wasm.md:113]: **issue:** This could be written with greater clarity. 


```suggestion
two browser families should be able to run
```

File: src/platform-integration/web/wasm.md
Comment (atsansone) [inline src/platform-integration/web/wasm.md:117]: **issue:** Google uses `later` in this case.


```suggestion
  - Version 119 or later.
```

File: src/platform-integration/web/wasm.md
Comment (atsansone) [inline src/platform-integration/web/wasm.md:119]: **issue:** Google uses `later` in this case.


```suggestion
  - Version 120 or later.
```

File: src/platform-integration/web/wasm.md
Comment (atsansone) [inline src/platform-integration/web/wasm.md:120]: **nit:** You should remove `currently`. All of our docs should be considered current, so you don't need this modifier.

**Review by atsansone — CHANGES_REQUESTED** [review]
Some changes needed for clarity.

File: src/platform-integration/web/wasm.md
Comment (parlough) [reply src/platform-integration/web/wasm.md:113]: Nice wording!

File: src/platform-integration/web/wasm.md
Comment (parlough) [reply src/platform-integration/web/wasm.md:120]: Good point.

This will change very soon or could change at any time, so I tried a different wording. Let me know what you think.

**Comment by parlough** [comment]
Thanks for the reviews!

@atsansone Great suggestions. I applied your changes with a few adjustments, let me know what you think :)

**Comment by domesticmouse** [comment]
119 is still a couple of days from stable, yeah?

**Comment by parlough** [comment]
Hmm, you're right, thanks. I didn't know they had a 1 week early "early stable" since 110. We can hold off until the 31st :)

**Comment by kevmoo** [comment]
Are we set up to land this? What's left??

**Comment by parlough** [comment]
I need an approval from @atsansone after his requested changes.

File: src/platform-integration/web/wasm.md
Comment (atsansone) [inline src/platform-integration/web/wasm.md:20]: **suggestion:** This reads a little awkward.
```suggestion
  Flutter is previewing these features.
  Ongoing development might result in frequent changes.
  Revisit this page for the latest updates.
```

**Review by atsansone — APPROVED** [review]
LGTM with one suggestion.

**Comment by parlough** [comment]
Somehow my “squash and merge” button switched to “rebase and merge” and I didn’t notice, so the merge for this PR wasn’t so clean…

I believe that option was previously wasn’t previously enabled, so I’m not sure what happened.

**Pull request merged**
